packages/compiler/test/decorators/decorators.ts

```diff
@@ -49,6 +49,57 @@ describe("compiler: built-in decorators", () => {
       strictEqual(getDoc(runner.program, B), "Templated B");
     });
 
+    it("applies @doc on all language elements", async () => {
```

Comment:
Not a major deal but it is good practice to test a single thing per `it`. Make is much more meaningful(Actually describe the expected behavior instead of acting like a smoke test) and easier to debug if something fails.
Like we already have "applies @doc on model" above.